### DISCUSSION TITLE
How to set --cleanup-local-files in Docker Environment Variables?

### DISCUSSION THREAD

**Discussion by chenjt2001** [abraunegg/onedrive#2863] [General]
Hello, how do I add parameter "--cleanup-local-files" to docker environment variables? I checked the [supported-docker-environment-variables](https://github.com/abraunegg/onedrive/blob/master/docs/docker.md#supported-docker-environment-variables), but there seems to be no this option. I need to use this with ONEDRIVE_DOWNLOADONLY=1.
I checked [#2708](https://github.com/abraunegg/onedrive/discussions/2708), but he was using version 2.4 and didn't need to add this option.

**Comment by abraunegg** [comment]
@chenjt2001 
You have 2 options:

1. Raise a Feature Request for this capability to be added as a Docker environment variable
2. Edit your Docker configuration to add `cleanup_local_files = "true"` to your 'config' file
    * Please read: https://github.com/abraunegg/onedrive/blob/master/docs/docker.md#editing-the-running-configuration-and-using-a-config-file on how to create and/or edit this file
    * Please read: https://github.com/abraunegg/onedrive/blob/master/docs/application-config-options.md for all configuration options

**Comment by chenjt2001** [comment]
> Edit your Docker configuration to add cleanup_local_files = "true" to your 'config' file

That solved my problem. Thank you for your efforts on this project. @abraunegg
